scripts/checkParentDenormalization.mjs

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable max-statements */
+/* eslint-disable no-continue */
+/* eslint-disable no-await-in-loop */
+/*
+Run this script simply with node, from the project root folder: node ./scripts/checkParentDenormalization.mjs
```

Comment:
I haven't tried yet, but just a comment: If this is useful enough to keep, we should ensure that it works with yarn as a traditional script.